public/app/features/dashboard/components/PanelEditor/OptionsPaneOptions.tsx

```diff
@@ -130,9 +131,9 @@ export const OptionsPaneOptions = (props: OptionPaneRenderProps) => {
         )}
       </div>
       <div className={styles.scrollWrapper}>
-        <CustomScrollbar autoHeightMin="100%">
+        <ScrollContainer minHeight="100%">
```

Comment:
scrollwrapper within a scrollwrapper? :rofl:

Reply:
yeah but i'm trying to avoid that 😄
in this case, i don't think the wrapper is even necessary.